content/authorization.md

```diff
@@ -74,20 +74,41 @@ register applications. Obtaining a Cerner Care account is free and
 requires nothing but an email address.
 
 If registering an application that is using the public app profile and requires access only when the user is online, 
-your client application must first register using our [code console][CERNER-CODE-CONSOLE]. 
+your client application must first register using our [code Console][CERNER-CODE-CONSOLE]. 
 
 If registering a confidential client application (required for offline_access and
-access on behalf of a system), please first request an account via the **non-production** instance of 
-[Cerner Central System Account Registration][SYSTEM-ACCOUNT-REGISTRATION]. Once the request is completed, use the
-issued system account GUID when registering the application in the [code console][CERNER-CODE-CONSOLE]. Be aware that
-access on behalf of a system currently has limited availability. Check the respective FHIR<sup>®</sup> implementation
-documentation to determine availability.
+access on behalf of a system), follow the directions in the [Registering a System Account][SYSTEM-ACCOUNT-SECTION] 
+section. Check the respective FHIR<sup>®</sup> implementation documentation to determine availability.
 
 Once registered, the **client identifier** will be provided for use with 
 the Cerner Authorization Server. As a registered client, Cerner 
 organizations may then ask for your client application to be enabled, 
 which is necessary in order to gain access to their protected resources.
 
+### Registering a System Account ###
+
+If an application will be using the offline_access scope, or accessing data on behalf of a system, it will need to
+maintain a secret. Currently, our implementation provides management and rotation functionality for this workflow in our 
+System Accounts application. In order to register one of these applications, you must first request a system account,
+and then register that as a SMART or FHIR application in our code Console.
+
+Request a system account for *non-production* by following these steps:
+
+- Login to the Cerner Central System Accounts application for non-production: https://sandboxcernercentral.com/system-accounts
+  - If you are a Cerner client using [My Domain][CERNER-CENTRAL-MYDOMAIN] for Cerner Central, log into your 
+non-production Cerner Central SSO account and select "System Accounts"
+- Fill out the fields as follows:
+  - **Description**: Representing a SMART/FHIR application used for... (name your application and company)
+  - **Production System**: No
+  - **Cerner Client**: No, unless you are a Cerner Client developing their own application. If you are a client:
+    - **Client Name**: Your organization. For example: My Health System
+    - **Client Number**: Only fill out if you know this, it is not required.
+    - **Client Mnemonic**: Only fill out if you know this, it is not required.  
+  - **Millennium System**: No
+- Once the account is created, take the id from that account and use it in the GUID field when you register the
```

Comment:
You could perhaps add a disclaimer here:

Note that requests for system accounts currently undergo manual review, and may therefore take a few business days to be created.

Reply:
I can add the 2 day TAT - though sandbox accounts don't have a review, I think it's mostly the queue that takes time. Well, assuming they can identify them as SMART/FHIR when requested (part of the reason we're defining what to put in description)